www/authoring/author/activity-edit.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+include_once '../../config.php';
+include_once 'includes/header.php';
+
+use LearnositySdk\Request\Init;
+use LearnositySdk\Utils\Uuid;
+
+$security = [
+    'consumer_key' => $consumer_key,
+    'domain'       => $domain
+];
+
+$request = [
+    'mode'      => 'activity_edit',
+    'reference' => 'learnosity-activity-demo',
```

Comment:
Can this be modified to a NEW reference each time? The problem is, if someone modifies this one, then it will show for everyone else whereas we want it to be a new reference everything the page is hit, like it does for item edit.